src/librustc/ty/layout.rs

```diff
@@ -511,11 +511,10 @@ pub struct Struct {
     /// If true, the size is exact, otherwise it's only a lower bound.
     pub sized: bool,
 
-    /// Offsets for the first byte after each field.
-    /// That is, field_offset(i) = offset_after_field[i - 1] and the
-    /// whole structure's size is the last offset, excluding padding.
-    // FIXME(eddyb) use small vector optimization for the common case.
-    pub offset_after_field: Vec<Size>
+    /// Offsets for the first byte of each field.
+    pub offsets: Vec<Size>,
```

Comment:
Does eddyb's fixme still apply?
`// FIXME(eddyb) use small vector optimization for the common case.`